### ISSUE TITLE
Do a longer gradient descent

### ISSUE THREAD

**Issue by chris-code** [chris-code/master-thesis#2]
For well-working AEs, it would be nice to see how confident the network becomes.

For AEs that don't work well, the question is whether their confidence saturates after time or if they are just very slow.

**Comment by chris-code** [comment]
This has been done, with the result that the success rate of AE generation doesn't improve. However, a higher learning rate can lead to success. Do a run with higher learning rate to try and achieve higher network confidence, even if that means the images get visibly distorted.
